### PR TITLE
horizon/docs: Minor aesthetic, phrasing, and grammatical improvements

### PR DESCRIPTION
**Pull request by Shaptic**
Mostly created so that we can use the PR UI to comment on places where we can make improvements.

Summarized list of changes / their rationale:

  - Most Markdown renderers prefer paragraphs to be all on one line (let your
    editor do the wrapping) as opposed to with manual breaks. This can be
    especially problematic with additional formatting e.g. `[links broken over
    a newline](#)`, etc. 

    GitHub's renderer is solid, but better safe than sorry for portability.
    Dropping whitespace changes from the diff should help w/ viewing. 

  - Unifying usage of "Captive Core" or "Captive Stellar-Core" (consistent
    capitalization, hyphenation, etc.).

  - Drop the example in the `CAPTIVE_CORE_CONFIG_APPEND_PATH` and link to the
    below one, instead, to avoid them diverging or otherwise getting invalid.

  - Fixed a few comma splices and other punctuation pedantry

  - Prefer the `systemctl` method of managing Horizon

  - Specify syntax highlighting methods where possible (e.g. the Bash script)

### CODE REVIEW

**Review by tamirms — APPROVED** [review]

File: services/horizon/internal/docs/captive_core.md [hunk 1]
```diff
@@ -17,61 +17,44 @@ replacement: https://developers.stellar.org/docs/horizon-captive-core/
...
 
 ![Horizon with Remote Captive Core](HorizonWithRemoteCaptiveCore.png)
-    
-Captive Core completely eliminates all Horizon issues connected Stellar-Core's database but requires extra time to initialize the Stellar-Core subprocess.
 
-Captive Stellar-Core can be used in both reingestion (`horizon db reingest range`) and normal Horizon operation (`horizon` or `horizon serve`).
-In fact, using Captive Core to reingest historical data is considerably faster without Captive Core (i.e. filling in Core's database with `stellar-core catchup` followed by `horizon db reingest range`).
+Captive Core completely eliminates all Horizon issues connected Stellar-Core's database, but it requires extra time to initialize the Stellar-Core subprocess.
+
+Captive Stellar-Core can be used in both reingestion (`horizon db reingest range`) and normal Horizon operation (`horizon` or `horizon serve`). In fact, using Captive Core to reingest historical data is considerably faster without Captive Core (i.e. filling in Core's database with `stellar-core catchup` followed by `horizon db reingest range`).
```
Comment (bartekn) [inline services/horizon/internal/docs/captive_core.md:43]:
```suggestion
Captive Stellar-Core can be used in both reingestion (`horizon db reingest range`) and normal Horizon operation (`horizon` or `horizon serve`). In fact, using Captive Core to reingest historical data is considerably slower without Captive Core (i.e. filling in Core's database with `stellar-core catchup` followed by `horizon db reingest range`).
```

**Review by bartekn — APPROVED** [review]

**Comment by 2opremio** [comment]
Niiice!

**Review by 2opremio — APPROVED** [review]

File: services/horizon/internal/docs/captive_core.md (hunk 1 above)
Comment (Shaptic) [reply services/horizon/internal/docs/captive_core.md:43]: Wait, I just realized this sentence doesn't make sense: "using Captive Core to reingest is faster without Captive Core"? What's the right verbiage here, should it be "slower **than** without Captive Core"?

Comment (Shaptic) [reply services/horizon/internal/docs/captive_core.md]: Verbiage fixed to the above in d561cd25, but I'm not 100% sure this is correct? (cc @2opremio)

Comment (2opremio) [reply services/horizon/internal/docs/captive_core.md]: Looks good!

**Comment by 2opremio** [comment]
I guess this is ready to merge? Ship it!!

**Comment by 2opremio** [comment]
I also guess we should retarget the PR to master or onto a new 2.0 (non-beta) release branch.

**Comment by Shaptic** [comment]
@cla-bot check

**Pull request merged**